### DISCUSSION TITLE
Globally disable spinner on QImg component

### DISCUSSION THREAD

**Discussion by VIM-Arcange** [quasarframework/quasar#15421] [General - Components / Directives / etc] [answered]
By default, the QImg component displays a spinner when loading the image, unless you set the no-spinner property to false.
Is there a way to disable it globally without editing all the QImg components in our app?

**Comment by MilosPaunovic** [comment, marked as answer]
Take a look at these links below, might give you some ideas:

https://github.com/quasarframework/quasar/issues/7379
https://github.com/quasarframework/quasar/discussions/8761

**Comment by VIM-Arcange** [comment]
Thank you for the hints. The proposed ideas look a bit hacky to me but will do the trick.
